### PR TITLE
Add balena UUID and balena fleet name to sentry tag

### PR DESCRIPTION
**Pull request by posterzh**
**Issue**

- Link: https://github.com/NebraLtd/hm-diag/issues/405
- Summary:
Add balena device UUID and balena app name(fleet name) to senty issues as a tag. 
Sentry tag is very convenient for filtering, tag-distribution maps, the frequency and the last time that Sentry has seen a tag.
The primary benefit of the Balena device UUID tag is that it enables further analysis of the Sentry problem on the Balena dashboard.

**How**
- Inside the balena container, get the balena UUID from environment variable `BALENA_DEVICE_UUID` and get balena app name from environment variable `BALENA_APP_NAME`.
- Set sentry tag with balena UUID and balena app name respectively.

**Screenshots**

**References**

**Checklist**

- [ ] Tests added
- [ ] Cleaned up commit history (rebase!)
- [ ] Documentation added
- [ ] Thought about variable and method names

### CODE REVIEW

**Comment by posterzh** [comment]
I confirmed `balena_id` and `balena_app` is in the sentry issues as tags by manually triggering exceptions on my miners.

File: hw_diag/utilities/sentry.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    """
+    Initialize sentry with balena_id and balena_app as tag.
+    If sentry_dsn is not set, do nothing.
+    """
+
+    if not sentry_dsn:
+        return
+
+    sentry_logging = LoggingIntegration(
+        level=logging.CRITICAL,
```
Comment (pritamghanghas) [inline hw_diag/utilities/sentry.py:19]: we can go back to error logging. We had decided sometime back that rate limiting is better than silencing errors level

Comment (posterzh) [reply hw_diag/utilities/sentry.py:19]: I didn't change the log level in this PR. We can change the level in another PR.

Comment (posterzh) [reply hw_diag/utilities/sentry.py:19]: Or we can do it here.

Comment (posterzh) [reply hw_diag/utilities/sentry.py]: Updated!

**Review by pritamghanghas — APPROVED** [review]

**Pull request merged**